airbyte-workers/src/test/java/io/airbyte/workers/TestConfigHelpers.java

```diff
@@ -82,33 +82,25 @@
         .withDestinationId(destinationId)
         .withTombstone(false);
 
-    final Field field = new Field()
-        .withName(FIELD_NAME)
-        .withDataType(DataType.STRING)
-        .withSelected(true);
-
-    final Stream stream = new Stream()
-        .withName(STREAM_NAME)
-        .withSelected(true)
-        .withFields(Collections.singletonList(field));
-
-    final Schema schema = new Schema().withStreams(Collections.singletonList(stream));
+    final ConfiguredAirbyteStream stream = new ConfiguredAirbyteStream()
+        .withStream(CatalogHelpers.createAirbyteStream(STREAM_NAME, Field.of(FIELD_NAME, JsonSchemaPrimitive.STRING)));
+    final ConfiguredAirbyteCatalog catalog = new ConfiguredAirbyteCatalog().withStreams(Collections.singletonList(stream));
 
     final StandardSync standardSync = new StandardSync()
         .withConnectionId(connectionId)
         .withDestinationId(destinationId)
         .withSourceId(sourceId)
         .withStatus(StandardSync.Status.ACTIVE)
         .withName(CONNECTION_NAME)
-        .withSchema(schema);
+        .withSchema(catalog);
```

Comment:
this should probably be called catalog instead of schema. i'm sympathetic that changing these can be a bit of a pain, so if this is really awful to fix don't worry about it. but since we're tearing everythhing up now, this is as good a time as any.

Reply:
Don't worry, it's not as painful as changing completely the object underneath haha

Reply:
I don't think it should affect him since this model is only used behind the API. the only thing that should affect him are changes to the API interface (in config.yaml).